README.md

```diff
@@ -83,7 +83,7 @@ kms_froala_editor:
 ### Step 4: Load Twig form widget
 
 ```yaml
-# In config/packages/twig.yaml or config/packages/kms_froala_editor.yaml
+# In config/packages/twig.yaml
```

Comment:
The docs are following the recipe here, that's why there are both paths.

Reply:
@jmsche  The file wasn't present by default in symfony project so removed `or config/packages/kms_froala_editor.yaml`.

Reply:
It's added by the recipe: https://github.com/symfony/recipes-contrib/blob/master/kms/froala-editor-bundle/3.0/config/packages/kms_froala_editor.yaml
And IMO it makes more sense to have a separate file for the bundle config